x/tradeshield/client/cli/tx_pending_spot_order.go

```diff
@@ -94,3 +96,46 @@ func CmdDeletePendingSpotOrder() *cobra.Command {
 
 	return cmd
 }
+
+func CmdDCancelSpotOrders() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cancel-pending-spot-orders [ids]",
```

Comment:
@cryptokage1996 can you add an example of how the CLI is used and the result of the CLI

Reply:
@cryptokage1996 I can’t see any examples in the CLIs